pkg/endpoint/endpoint.go

```diff
@@ -833,7 +833,8 @@ func (e *Endpoint) Update(cfg *models.EndpointConfigurationSpec) error {
 	// Note: This "retry" behaviour is better suited to a controller, and can be
 	// moved there once we have an endpoint regeneration controller.
 	regenCtx := &regeneration.ExternalRegenerationMetadata{
-		Reason: "endpoint was updated via API",
+		Reason:            "endpoint was updated via API",
+		RegenerationLevel: regeneration.RegenerateWithoutDatapath, // Needs datapath load??
```

Comment:
The code immediately below this statement already handles the cases where we're worried about a higher regeneration level, so this can probably regenerate without the datapath. (Assuming the `updateAndOverrideEndponitOptions()` does what it sounds like).